### PR TITLE
Add image including .NET 7

### PR DESCRIPTION
**Pull request by pascalberger**
Add image including .NET 7

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,61 @@
...
+      - task: SonarCloudAnalyze@1
+        displayName: 'Run SonarCloud analysis'
+        target: swissgrc/azure-pipelines-sonarscannermsbuild:latest
+```
+
+### Tags
+
+| Tag          | Description                                     | Base Image                                | .NET SDK | NodeJS  | Git                | Git LFS | Size                                                                                                                                           |
+|--------------|-------------------------------------------------|-------------------------------------------|----------|---------|--------------------|---------|------------------------------------------------------------------------------------------------------------------------------------------------|
+| unstable     | Latest unstable release (from `develop` branch) | swissgrc/azure-pipelines-openjdk:17.0.7.0 | 7.0.302  | 18.16.0 | 1:2.39.2-1~bpo11+1 | 3.3.0   | ![Docker Image Size (tag)](https://img.shields.io/docker/image-size/swissgrc/azure-pipelines-sonarscannermsbuild/6-unstable?style=flat-square) |
```
Comment (Xzelsius) [inline README.md]: Do we remove the unstable tag once this PR is merged?

**Review by Xzelsius — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (pascalberger) [reply README.md]: @Xzelsius I would keep `unstable` for pointing to the unstable release of the latest .NET version.

Comment (Xzelsius) [reply README.md]: Means we would need to adjust the workflow that sets the tags. As you specifically set the unstable tag to another name, so there wont be a new unstable tag.

Comment (pascalberger) [reply README.md]: Got it. Will remove the unstable tag from README and remove the tag after merging

**Review by Xzelsius — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -1,2 +1,60 @@
...
+      - task: SonarCloudAnalyze@1
+        displayName: 'Run SonarCloud analysis'
+        target: swissgrc/azure-pipelines-sonarscannermsbuild:latest
+```
+
+### Tags
+
+| Tag          | Description                                     | Base Image                                | .NET SDK | NodeJS  | Git                | Git LFS | Size                                                                                                                                           |
+|--------------|-------------------------------------------------|-------------------------------------------|----------|---------|--------------------|---------|------------------------------------------------------------------------------------------------------------------------------------------------|
+| 7-unstable   | Latest unstable release (from `develop` branch) | swissgrc/azure-pipelines-openjdk:17.0.7.0 | 7.0.302  | 18.16.0 | 1:2.39.2-1~bpo11+1 | 3.3.0   | ![Docker Image Size (tag)](https://img.shields.io/docker/image-size/swissgrc/azure-pipelines-sonarscannermsbuild/6-unstable?style=flat-square) |
```
Comment (Xzelsius) [inline README.md]: 6-unstable?

**Review by Xzelsius — CHANGES_REQUESTED** [review]

**Review by Xzelsius — APPROVED** [review]

**Pull request merged**